src/js/components/blog/BlogArticle.js

```diff
@@ -0,0 +1,52 @@
+import React, { PropTypes } from 'react';
+import SocialLinks from './SocialLinks';
+import {  Article } from './mockData';
+
+class BlogArticle extends React.Component {
+  constructor() {
+    super();
+    this.setArticleBody = this.setArticleBody.bind(this);
+  }
+
+  componentDidMount() {
+    const parser = new DOMParser();
+    const htmlDoc = parser.parseFromString(Article.body, "text/html");
+    this.articleBody.innerHTML = htmlDoc.body.innerHTML;
+  }
+
+  setArticleBody(articleBody) {
```

Comment:
To avoid needing the explicit `bind` in the constructor,  you could declare the `setArticleBody` method as a property of the `BlogArticle` class instance.

```
setArticleBody = (articleBody) => {
  // stuff
}
```